src/main/kotlin/com/papsign/ktor/openapigen/SwaggerUI.kt

```diff
@@ -14,15 +14,15 @@ import io.ktor.request.port
 import io.ktor.response.respond
 import java.net.URL
 
-class SwaggerUi(val basePath: String) {
+class SwaggerUi(private val basePath: String, private val version: String) {
```

Comment:
Technically, not backward compatible, but I doubt anyone uses these properties.

Reply:
Maybe we could add the packaged swaggerUI version as the deafult version value, so we don't even have to worry about that.

Reply:
What I meant is about making the properties private.

And anyway, the current release is still snapshot, so probably people should expect these sort of issues until a stable release.

I'd say keep this like so, instead of risking questionable design choices. What do you think?